docs/HowTo/Get-Started/System-Requirements.md

```diff
@@ -30,6 +30,24 @@ To determine system requirements, check CPU and disk space requirements using
 Besu requires 4GB of RAM. For public Ethereum networks, including MainNet and testnets such as
```

Comment:
You should either remove or modify this sentence that contradicts your new text

Reply:
#275 also makes this changes on requirements, so I think it has nothing to do in this PR. Please fix.

Reply:
Ugggh! I created this branch off of the branch from the previous PR instead of master! I think it will be simpler to close this PR and redo another. The changes are small enough.